api/test/robot_tests/plugin_manager/plugin_manager.robot

```diff
@@ -0,0 +1,183 @@
+*** Settings ***
+Documentation       This is a test suite for Thyra Plugin Manager endpoints.
+
+Library             RequestsLibrary
+Library             String
+Resource            keywords.resource
+Resource            ../variables.resource
+
+Suite Setup         Suite Setup
+
+
+*** Test Cases ***
+GET /plugin-manager with no plugins
+    ${response}=    GET    ${API_URL}/plugin-manager
+    Status Should Be    ${STATUS_OK}
+    ${listLength}=    Get Length    ${response.json()}
+    Should Be Equal As Integers    ${listLength}    0
+
+POST /plugin-manager?source={{pluginSource}}
+    ${source}=    Set Variable
+    ...    https://github.com/massalabs/thyra-plugin-hello-world/releases/download/0.0.3/thyra-plugin-hello-world_${OS}-${ARCH}.zip
+    ${response}=    POST
+    ...    ${API_URL}/plugin-manager
+    ...    params=source=${source}
+    ...    expected_status=${STATUS_NO_CONTENT}
+    Sleep    1 seconds    # Wait for the plugin to be registered
+
+GET /plugin-manager with one plugin
+    ${response}=    GET    ${API_URL}/plugin-manager
+    Status Should Be    ${STATUS_OK}
+    ${listLength}=    Get Length    ${response.json()}
+    Should Be Equal As Integers    ${listLength}    1
+
+GET /plugin-manager/{id}
+    ${response}=    GET    ${API_URL}/plugin-manager
+    ${id}=    Set Variable    ${response.json()[0]['id']}
+    ${response}=    GET    ${API_URL}/plugin-manager/${id}
+    Status Should Be    ${STATUS_OK}
+    Should Be Equal As Strings    ${response.json()['status']}    Up
+
+POST /plugin-manager/{id}/execute with stop command
+    ${id}=    Get Plugin ID From Author and Name    massalabs    hello world
+    ${data}=    Create Dictionary    command=stop
+    ${response}=    POST
+    ...    ${API_URL}/plugin-manager/${id}/execute
+    ...    expected_status=${STATUS_NO_CONTENT}
+    ...    json=${data}
+
+POST /plugin-manager/{id}/execute with start command
+    ${id}=    Get Plugin ID From Author and Name    massalabs    hello world
+    ${data}=    Create Dictionary    command=start
+    ${response}=    POST
+    ...    ${API_URL}/plugin-manager/${id}/execute
+    ...    expected_status=${STATUS_NO_CONTENT}
+    ...    json=${data}
+    Sleep    1 seconds    # Wait for the plugin to be started
+
+POST /plugin-manager/{id}/execute with restart command
+    ${id}=    Get Plugin ID From Author and Name    massalabs    hello world
+    ${data}=    Create Dictionary    command=restart
+    ${response}=    POST
+    ...    ${API_URL}/plugin-manager/${id}/execute
+    ...    expected_status=${STATUS_NO_CONTENT}
+    ...    json=${data}
+    Sleep    1 seconds    # Wait for the plugin to be restarted
+
+GET /thyra/plugin/{author}/{name}/
+    ${response}=    GET
+    ...    ${API_URL}/thyra/plugin/massalabs/hello%20world/
+    ...    expected_status=${STATUS_OK}
+    Should Contain    ${response.text}    Hello, world!
+
+# We can register multiple times the same plugin, but the aliases list isn't updated correctly.
```

Comment:
You should add a link tho the bug fix issue here

Reply:
Done https://github.com/massalabs/thyra/pull/564/commits/71ef22d6f659639c8624069b2b75b18aa2ede750